test/Microsoft.VisualStudio.Web.CodeGeneration.MSBuild.Test/Microsoft.VisualStudio.Web.CodeGeneration.MSBuild.Test.csproj

```diff
@@ -13,7 +13,7 @@
 
   <ItemGroup>
     <ProjectReference Include="..\..\src\Microsoft.VisualStudio.Web.CodeGeneration.Msbuild\Microsoft.VisualStudio.Web.CodeGeneration.Msbuild.csproj" />
-    <PackageReference Include="Microsoft.Build.Runtime" Version="15.1.548" />
+    <PackageReference Include="Microsoft.Build.Runtime" Version="15.2.0-preview-000047-02" />
```

Comment:
Could you move these versions to the dependencies.props? Easier to find and replace if we ever need to bulk update them.